### PR TITLE
[Feature] CX-5282 Convert GenericError component

### PR DESCRIPTION
**Pull request by josokinas**
Converting GenericError component.

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1090-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1090-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1090-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/GenericError/style.scss [hunk 1]
```diff
@@ -1,23 +1,23 @@
-@import (less) "../Theme/constants.css";
+@import '../Theme/constants';
 
-.icon {
-  height: 144*@unit;
+%icon {
```
Comment (kopijunkie) [inline src/components/GenericError/style.scss:3]: Minor: should we use the _ for internal only things like this here too?

Comment (josokinas) [reply src/components/GenericError/style.scss:3]: Current configuration does not allow this 🙈 Rule is only set for mixins, as they behave like functions... I'll leave this like this for now, feel free later to adjust.

**Pull request merged**